gui/src/components/pages/TaipyRendered.tsx

```diff
@@ -107,7 +109,7 @@ const TaipyRendered = (props: TaipyRenderedProps) => {
                     })
                 );
         }
-    }, [path, state.id, dispatch, partial, fromBlock, baseURL]);
+    }, [path, state.id, dispatch, partial, fromBlock, baseURL, location.search]);
```

Comment:
a bit worried about the possible side effects of location.search dependencies ...